packages/contracts-bedrock/scripts/deploy/DeploySuperchain.s.sol

```diff
@@ -340,85 +412,128 @@ contract DeploySuperchain is Script {
         _dso.set(_dso.superchainProxyAdmin.selector, address(superchainProxyAdmin));
     }
 
```

Comment:
Going to need review from @maurelian @mds1 on the deploy script modifications here